README.rst

```diff
@@ -48,10 +48,10 @@ index page is `here <http://pypi.python.org/pypi/pyFFTW>`_.
 
 Requirements (i.e. what it was designed for)
 --------------------------------------------
-- Python 2.7 or greater (Python 3 is supported)
-- Numpy 1.6
+- Python 2.7 or greater (Python 3.4+ is also supported)
+- Numpy 1.10
 - FFTW 3.3 or higher (lower versions *may* work)
-- Cython 0.15 or higher (though the source release on PyPI loses this
+- Cython 0.23 or higher (though the source release on PyPI loses this
```

Comment:
Just curious if there were any known issues this would be fixing. Certainly in favor of bumping Cython. Only wondering if there is anything else motivating it.

Reply:
I just updated this text test to match what is currently be in our `setup.py`.  I am not sure exactly what the reason for 0.23 specifically was, but perhaps @fredRos may recall (I believe he added this version check in one of his recent PRs).  

https://github.com/pyFFTW/pyFFTW/blob/f4bdbfcad95b42fb1b0705c084d17f2d39d1d0a7/setup.py#L757